string traj_status
int32 current_traj

---

bool success